contents/blog/the-posthog-array-1-39-0.mdx

```diff
@@ -0,0 +1,128 @@
+
+### Improved: Turbo properties
+
+Frustrated that it was taking a few seconds to load properties? You weren't the only one. 
+
+Thankfully, version 1.39.0 addresses this by adding a limit so that PostHog doesn't always search through hundreds of thousands of properties for each query. The result? [Properties now load much faster](https://github.com/PostHog/posthog/pull/11037). On PostHog Cloud we saw a 75% reduction in API call time. 
+
+### Improved: History log now shows changes to insights
+
+![a view of the insights activity tab](../images/blog/array-1.39.0/insights-activity.png)
+
+We've had feedback from several users that they enjoy tracking changes to their feature flags in the history logs. So, we added the same functionality to insights too! Now you can snoop on who has been changing your insights and giving them funny names.
+
+## Improved: Client and server-side library improvements
+
+While we continue to recommend using Javascript if possible, we've made significant updates to our other client libraries so that Android, iOS, Flutter and React Native now all support feature flags. We've also updated our server-side libraries to have local feature flag evaluation.
+
+Check [our library docs](/docs/integrate/libraries) to find out more about what each library supports and how to deploy. 
+
+### Improved: Automatically comment SQL queries
+
+We've added [a new SQL commenter dependency](https://google.github.io/sqlcommenter/python/django/#0) which automatically comments all SQL queries generated by PostHog with tags. We've tried this twice before with [different solutions](https://github.com/PostHog/posthog/pull/10433#issuecomment-1164125629), but we're confident that _this_ time will definitely be the third time we've tried. 
+
+### Apps: Census integration
+
+Census, a reverse ETL platform which syncs your data warehouse with your operational tools, has created a new PostHog integration. Now you can easily sync events data such as sales or marketing activity into PostHog. Check [the Census docs](https://docs.getcensus.com/destinations/posthog) to find out more!
+
+Interested in building your own app? [Here's how to get started](/docs/apps)!
+
+### Other improvements & fixes
+Version 1.39 also adds hundreds of other improvements and fixes, including...
+
+- **Improvement**: Async migrations that aren't available yet will show up in a separate tab and migrations in Starting state can now be stopped in the UI. This solves [a few problems](https://github.com/PostHog/posthog/pull/10912).
+-   **Improvement**: We've removed some axe violations on Home, to improve accessibility. And added accessibility testing to our workflow. Bonus: Check the PR for [a _fantastic_ Dad joke](https://github.com/PostHog/posthog/pull/11211#discussion_r941651022). 
+-   **Improvement**: When deleting persons, you can now choose to delete event data as well thanks to [a new person deletion flow](https://github.com/PostHog/posthog/pull/11347).
```

Comment:
This should maybe be higher/it's own section? GDPR compliance etc

Reply:
+1